src/aura/STG_CMP_Tabs/STG_CMP_TabsHelper.js

```diff
@@ -204,6 +207,38 @@
         $A.enqueueAction(action);
     },
 
+    getFluencyPicklistEntries : function(component, fluencyValue) {
```

Comment:
@robertbwatson -- I am not sure why this is showing as "new." I had conflicts, so merged the diffs from master to bring in the fluency settings logic. All I can think of was that switching things from tabs to spaces just freaked out git?